src/quarks/StyledInputRow.tsx

```diff
@@ -49,7 +49,7 @@ const TextFieldLabelText = styled('span')<TextFieldLabelProps>`
   background: ${themed(({ multiline, theme }) => (multiline ? theme.ui2 : 'transparent'))};
 
   > span {
-    opacity: ${props => (props.active || props.filled ? '0' : '1')};
+    display: ${props => (props.active || props.filled ? 'none' : '')};
```

Comment:
I'd prefer to use `visibility` in this case.

Can you try instead of `display` to use `visibility: hidden|visible`?

Reply:
Using `visibility` won't work, because we need to make sure the inner `span` isn't used for calculating the width of the label.
I tried it with `visibility: hidden`, but the only thing working as intended was `display: none`.

Reply:
Gotcha. Thanks for the explanation.